bin/node/runtime/src/lib.rs

```diff
@@ -546,6 +546,13 @@ impl pallet_staking::BenchmarkingConfig for StakingBenchmarkingConfig {
 	type MaxValidators = ConstU32<1000>;
 }
 
+impl pallet_stake_tracker::Config for Runtime {
+	type Currency = Balances;
+	type Staking = Staking;
+	type VoterList = VoterListTracker;
+	type TargetList = TargetList;
```

Comment:
the names in two lines are not consistent -- either both `somethingTracker` or neither.

Reply:
The idea here is that the TargetList is not used in Staking yet, so we can start tracking it in staking-tracker and then passing it to Staking without any migrations.
Since VoterList already exists - I had to add a suffix.